docs/setup-and-configuration.md

```diff
@@ -11,5 +11,486 @@ A client is usually some system that you want to able to send request to the Ope
 
 ### What is a Channel?
 
-A Channel defines a path that a request will take through the OpenHIM. It describes one or more routes for the request to be forwarded to, which clients are allowed to use the channel, which requests are to be directed to this channel and many more options that allows you to control what happens for a particular request. 
+A Channel defines a path that a request will take through the OpenHIM. It describes one or more routes for the request to be forwarded to, which clients are allowed to use the channel, which requests are to be directed to this channel and many more options that allows you to control what happens for a particular request.
 The management of clients and channels are discussed later in the document. Only an OpenHIM administrative user has the permission to Add, Edit and Delete a Client or Channel
+
+## OpenHIM Clients
+
+Using an OpenHIM administrative account, you will be able to add, edit and remove clients by following a few easy steps.
+
+The following is an explanation of the fields that are used in the Add Client form:
+- **Client ID** - This is a unique ID which a client will use as a reference when adding channels as well as for authorisation checking.
+- **Client Name** - This is a descriptive name of the client.
+- **Domain** - A domain that is associated with a client. 
+    > **Note**: The domain needs to match the CN of the certificate if a certificate is used otherwise the client won’t be authorised successfully
+- **Roles** - The client roles field is a list of authorized user groups that are allowed to access this channel. You can either select a role from the suggested roles that appear when you start typing, or you can add a new role to the list by typing in the role and pressing “Enter“. 
+    > **Note**: suggested roles will only appear as you type, if they already exist in the OpenHIM.
+- **Certificate** - The certificate field is used when the OpenHIM core is running using mutual TLS authentication and needs to authenticate requests coming from the client. 
+By default, the OpenHIM core uses mutual TLS authentication
+- **Basic Auth Password** - The password field is used when the OpenHIM core is running in basic auth mode and does not require a certificate, it does however require a password.
+
+### How to add clients
+> **Note**: All fields marked with a * or ** indicates a mandatory field.
+
+    * - Indicates a required field which means that it cannot be left blank
+    ** - Indicates that one of the fields are required
+
+1. Logon to your OpenHIM console
+1. Click on Clients found in the left navigation menu.
+1. Click on the button labelled “+ Client” to open a popup window where you will be able to supply the client details.
```

Comment:
There is an inconsistency within the different lists - some lists elements end in full stops some don't. Please choose one style and apply to all the lists